CancerGov/_src/StyleSheets/_tableSort.scss

```diff
@@ -55,62 +55,58 @@ table[data-sortable] td[data-sorted="true"]{
   table[data-sortable] {
     height: 100% !important;
   }
-  
-  table[data-sortable] tfoot tr td:last-child {
-    padding-bottom: 20px;
-    background-color: #ace;
-  }
 
-  table[data-sortable] thead tr th:first-child,
-  table[data-sortable] tbody tr td:first-child
-  /*,
-  table[data-sortable] tfoot tr td:first-child*/ {
-  //background: #ace;
-  top: auto;
-  left: 0.5px;
-  position: absolute !important;
-  width: 160px !important;
-  z-index: 3;
-  }
   table[data-sortable] thead tr th:first-child {
     //background-color: #ace;
     height: 7.85em;
-  }
-  table[data-sortable] tbody tr td:first-child {
-    background-color: #fff;
-    //left: -1px;
-    height: 200px !important;
-    line-height: 1.25em;
+    box-shadow: 0 2px 12px 3px rgba(0,0,0,.2);
   }
 
-  table[data-sortable] tbody tr td, 
-  /*table[data-sortable] tfoot tr td,*/ 
-  table[data-sortable] tr td {
-    height: 200px !important;
-    //padding: 20px;
-    //height: 100% !important;
+  table[data-sortable] thead tr th[data-sorted="false"] {
+    background-color: #2b7bba;
   }
-  table[data-sortable] tfoot tr td {
-    height: auto !important;
+  
+  table[data-sortable] thead tr th:first-child,
+  table[data-sortable] tbody tr td:first-child {
+    //background: #ace;
+    border-right: 1px solid #ccc;
+    border-left: 1.25px solid #ccc;
+    border-bottom: none;
+    top: auto;
+    left: 0px;
+    position: absolute !important;
+    width: 160px !important;
+    z-index: 3;
+  }
+  
+  table[data-sortable] tbody tr:first-child td:nth-child(1) {
+    //background-color: #306;
+    height: calc(100% - 125px) !important;
+    box-shadow: 0 11px 12px 3px rgba(0,0,0,.2);
   }
 
-  table[data-sortable] thead tr th:first-child, 
   table[data-sortable] tbody tr td:first-child {
-      border-right: 1px solid #ccc;
-      box-shadow: 0px 11px 12px 3px rgba(0, 0, 0, 0.2);
-      left: 0px !important;
-      border-bottom: none;
+    background-color: #fff;
+    line-height: 1.25em;
   }
+
   // to show second column behind the first
   table[data-sortable] thead tr th:nth-child(2),
   table[data-sortable] tbody tr td:nth-child(2) {
     padding-left: 175px;
   }
+
   table[data-sortable] tbody tr td[data-sorted="true"]:first-child {
     background-color: #fafafa;
   }
-  table[data-sortable] thead tr th[data-sorted="false"] {
-    background-color: #2b7bba;
```

Comment:
This should be added to _colors.scss

Reply:
#2B7BBA already existed in _colors.scss, so the hex value was replaced with the variable name.  Replaced #FAFAFA with existing value #F9F9F9 (with design approval).  Added  #175E95 to _colors.scss as variable name $endeavor, as it is an approved sortable tables active column color.